app/views/polls/show.html.erb

```diff
@@ -38,13 +38,21 @@
           <%= t("polls.show.already_voted_in_booth") %>
         </div>
       <% else %>
+
+      <% if !@poll.voters.find_by_user_id(current_user).present? %>
```

Comment:
Check `poll_voter_token` function, we should be checking as well `origin: 'web'` I think. Since Poll::Voter could have been created through booth. 

Also `!QUERY.present?` == `QUERY.blank?` and its more readable I think

I would go with `poll_voter_token(@poll, current_user).empty?` because if that method returns `''` means that there is no token already generated and we could be making this div visible nearly. That way logic still is kept on one single place.